base/precompile.jl

```diff
@@ -1114,28 +1114,28 @@ precompile(Tuple{typeof(Base._jl_spawn), String, Array{String, 1}, Ptr{Void}, Ba
 precompile(Tuple{Type{Base.Timer}, Int64, Float64})
 precompile(Tuple{typeof(Base.sleep), Int64})
 precompile(Tuple{typeof(Base._uv_hook_close), Base.PipeEndpoint})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{IO}}})
-precompile(Tuple{Type{Base.Nullable{IO}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{AbstractString}}})
-precompile(Tuple{Type{Base.Nullable{AbstractString}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Integer}}})
-precompile(Tuple{Type{Base.Nullable{Integer}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Union{Int64, Symbol}}}})
-precompile(Tuple{Type{Base.Nullable{Union{Int64, Symbol}}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Cmd}}})
-precompile(Tuple{Type{Base.Nullable{Base.Cmd}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Any}}})
-precompile(Tuple{Type{Base.Nullable{Any}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Bool}}})
-precompile(Tuple{Type{Base.Nullable{Bool}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Process}}})
-precompile(Tuple{Type{Base.Nullable{Base.Process}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Base.Dict{K, V} where V where K}}})
-precompile(Tuple{Type{Base.Nullable{Base.Dict{K, V} where V where K}}})
-precompile(Tuple{typeof(Base.eltype), Type{Base.Nullable{Array{T, N} where N where T}}})
-precompile(Tuple{Type{Base.Nullable{Array{T, N} where N where T}}})
-precompile(Tuple{typeof(Base.convert), Type{Base.Nullable{Base.Process}}, Base.Process})
-precompile(Tuple{typeof(Base.convert), Type{Base.Nullable{Bool}}, Bool})
+precompile(Tuple{typeof(Base.eltype), Type{Base.Union{Some{IO}, Void}}})
```

Comment:
They usually need to be separate. Although note that in the case it's passing `Type{Union{...}}`, so that's one value, not a Union of types, so it is correct as-is